.github/workflows/webviz-subsurface-components.yml

```diff
@@ -97,23 +97,73 @@ jobs:
           pytest ./tests --headless
 
       # Run cypress tests on firefox browser
-      # - name: E2E on firefox
-      #   if: github.event_name != 'release' && matrix.e2e-on-firefox
-      #   uses: cypress-io/github-action@33ec2f03412577367c7c6aac7b646c88e646e33c
-      #   with:
-      #     browser: firefox
-      #     working-directory: ./react
-      #     start: npm run storybook
-      #     wait-on: "http://localhost:6006"
-      # - uses: actions/upload-artifact@v2
+      - name: Cache Cypress
+        if: github.event_name != 'release' && matrix.e2e-on-firefox
+        id: cache-cypress
```

Comment:
Cypress needs to install Cypress Binary in cache to function properly . Installation with  --ignore-scripts,  prevents Cypress installing the binary. 
Previously, "cypress-io/github-action" action was used to take care of this issue.
But recently I noticed this action is unable to handle the issue.
[Issue posted here](https://stackoverflow.com/questions/69439269/the-cypress-npm-package-is-installed-but-the-cypress-binary-is-missing-in-gi)